libfwupd/fwupd-device.c

```diff
@@ -1612,6 +1612,92 @@ fwupd_pad_kv_ups (GString *str, const gchar *key, FwupdUpdateState value)
 	fwupd_pad_kv_str (str, key, fwupd_update_state_to_string (value));
 }
 
+/**
+ * fwupd_device_to_json:
+ * @builder: A #JsonBuilder
+ * @device: A #FwupdDevice
+ *
+ * Adds a fwupd device to a JSON builder
+ *
+ * Since: 1.2.6
+ **/
+void
+fwupd_device_to_json (JsonBuilder *builder, FwupdDevice *device)
```

Comment:
I think the args are flipped, you want device, builder -- no?

Reply:
fixed